sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Utilities.cs

```diff
@@ -98,17 +99,17 @@ public static HttpResponseMessage BuildErrorResponse(EventErrorResponse error)
             }
             else
             {
-                // JObject or string type, use string to convert between JObject and JsonDocument.
+                // JObject or string type, use string to convert between NewtonsoftJson and SystemTextJson.
                 originStr = response.ToString();
-                converted = JsonDocument.Parse(originStr);
+                jResponse = response is JObject jObj ? jObj : JObject.Parse(originStr);
             }
 
             try
             {
                 // Check error, errorCode is required for json convert, otherwise, ignored.
```

Comment:
`"code"` is sufficient to detect the presence of an error?  Is there more we could/should validate?  I worry about how this versions over time because it might not be obvious to someone someday in the future that adding a "code" property on one of the events will change this control flow significantly.

Reply:
Understand your concern and do make sense. Add a common `StatusCode` to reserve a field for converter JsonObject to C# types.